finance_dl/amazon.py

```diff
@@ -99,14 +103,21 @@ def login(self):
             lambda: self.find_visible_elements(By.XPATH, '//input[@type="email"]')
         )
         username.send_keys(self.credentials['username'])
-        username.send_keys(Keys.ENTER)
 
         logger.info('Looking for password link')
         (password, ), = self.wait_and_return(
             lambda: self.find_visible_elements(By.XPATH, '//input[@type="password"]')
         )
         password.send_keys(self.credentials['password'])
+
+        logger.info('Looking for "remember me" checkbox')
+        (rememberMe, ) = self.wait_and_return(
+            lambda: self.find_visible_elements(By.XPATH, '//input[@name="rememberMe"]')[0]
+        )
+        rememberMe.click()
+
```

Comment:
Is this for debugging?  In general the issue with remembering logins and using a persistent profile is that you need additional logic to determine whether you are already logged, in order to skip login, or does the sign in logic work even if already signed in?

Reply:
added login detection